src/main/java/org/gridsuite/network/map/dto/ElementInfos.java

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.powsybl.iidm.network.*;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.network.map.model.CurrentLimitsData;
+import org.gridsuite.network.map.model.TapChangerData;
+import org.gridsuite.network.map.model.TapChangerStepData;
+import org.gridsuite.network.map.model.TemporaryLimitData;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@SuperBuilder
+@Getter
+public class ElementInfos {
+
+    public enum InfoType { //LevelOfDetail
+        LIST,
+        MAP,
+        FORM,
+        TAB
+    }
+
+    private String id;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private String name;
+
+    public static ElementInfos toData(Identifiable<?> identifiable) {
+        ElementInfos.ElementInfosBuilder builder = builder()
+                .name(identifiable.getOptionalName().orElse(null))
+                .id(identifiable.getId());
+        return builder.build();
+    }
+
```

Comment:
I think all these following utility methods should be put in a separate utility class, and not in this dto

Reply:
Same remark as in my previous response.

Reply:
Done